session-management/session-management.js

```diff
@@ -10,21 +10,26 @@ const mongo_client = new MongoClient(url);
 var db;
 
 mongo_client.connect(function(err, client) {
-  assert.equal(null, err);
+    assert.equal(null, err);
     console.log("Connected correctly to server");
     db = mongo_client.db("session-management"); 
 })
 
-              
 
 Consumer = kafka.Consumer,
 kafka_client = new kafka.KafkaClient()
-consumer = new Consumer(kafka_client, [{ topic: "session-management"}])
+consumer = new Consumer(kafka_client, [{ topic: "sessionmanagement"}, { topic: "session-update"}])
 
 consumer.on("message", function(message) {
-  console.log(message);
+    console.log(message);
+    if(message.topic == "sessionmanagement") {
+        db.collection("sessions").insertOne(message)
+    }
+    else {
```

Comment:
hey im not clear about update, is it when user changes his input details?

Reply:
No its when the request is processed  by the bottom three services and we want to update the payload with result .

Reply:
okay got it thanks